src/runtime_src/hip/api/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # Copyright (C) 2023 Advanced Micro Device, Inc. All rights reserved.
 add_library(hip_api_library_objects OBJECT
   hipDeviceGet.cpp
+  hipMemory.cpp
```

Comment:
Please change the file name to hip_memory.cpp. Change name of hipDeviceGet.cpp also.
XRT follows underscore for function names and file names (with small letters)

Reply:
done.